### ISSUE TITLE
Kindly help provide an Synthetic example( with custom data) to get started for Seq2Seq Model 

### ISSUE THREAD

**Issue by raghavchalapathy** [awslabs/gluon-ts#521]
Hi 

Can you please help me provide the code for seq2seq model , I can see deepAR, SimpleFeedForwardNetwork in the tutorials

with regards
Raghav

**Comment by raghavchalapathy** [comment]
estimator = Seq2SeqEstimator(freq=str(Cfg.freq),
                                       prediction_length=Cfg.horizon,
                                       cardinality=[1],
                                       trainer=Trainer(epochs=int(Cfg.n_epochs)),
                                       use_feat_dynamic_real=bool(Cfg.use_feat_dynamic_real),
                                       use_feat_static_cat=bool(Cfg.use_feat_static_cat))

I am currently using it like this, 
self.best_deepState_model = estimator.train(training_data=train_ds)

Things I would like to achieve is 

1) I would like to have a different input sequence( english.txt) and different output sequence (Spanish.txt) 

2) Customize the hidden layers 

3) Add Custom attention 

4) Add Custom loss function 

Can I do this easily in gluonts Please help respond

Raghav

**Comment by mbohlkeschneider** [comment]
Hi @raghavchalapathy ,

> I would like to have a different input sequence( english.txt) and different output sequence (Spanish.txt)

Just to make this clear: GluonTS is a time series library, i.e. it the domain is real values. We cannot do translation between languages (like English and Spanish in your example). 

If you want to modify the model, you need to directly modify the model code and implement your own layers/attention/losses. Its hard to point you to a particular place.

Please let us know if you have a specific question and we try to help you out.

**Comment by mbohlkeschneider** [comment]
Closing this for now. Please reopen if you have further questions.
